musicrecs/templates/index.html

```diff
@@ -2,39 +2,41 @@
 {% import "bootstrap/wtf.html" as wtf %}
 
 {% block content %}
-    <div class="bg-light" style="padding: 15px; margin-top: 10px; margin-bottom: 10px;">
-        <h2 style="margin-bottom: 20px;">Welcome to Musicrecs!</h2>
+    <div class="bg-light div_main">
+        <h2>Welcome to Musicrecs!</h2>
         <p>Musicrecs is a platform for sharing spotify music recommendations anonymously with friends.</p>
         <p>To get started, <a href="#new_round_form">create a new music recommendation round</a> and then share the created round link with others.</p>
         <p>The recommendation round will have three phases:</p>
-        <p style="margin-left: 20px;"><i>Submit Phase</i>: Participants submit spotify music links of their recommendations.</p>
-        <p style="margin-left: 20px;"><i>Listen Phase</i>: Participants listen to the music and consider who may have submitted what.</p>
-        <p style="margin-left: 20px;"><i>Revealed Phase</i>: It is revealed which participants recommended what.</p>
+        <p class="p_index_phase"><i>Submit Phase</i>: Participants submit spotify music links of their recommendations.</p>
+        <p class="p_index_phase"><i>Listen Phase</i>: Participants listen to the music and consider who may have submitted what.</p>
+        <p class="p_index_phase"><i>Revealed Phase</i>: It is revealed which participants recommended what.</p>
```

Comment:
The text on this page looks too cramped to me now. Maybe your styling is revealing that having separate paragraphs for each line is not a good idea haha.

Reply:
![image](https://user-images.githubusercontent.com/13397301/108632818-daa56980-743e-11eb-9b9a-b701db2dfbc9.png)

Reply:
![image](https://user-images.githubusercontent.com/13397301/108633087-5b189a00-7440-11eb-80ab-a284ba39ad56.png)

Reply:
But now that I'm looking at how it compares to the old one, that old one is too spaced out. I think we need a goldilocks spacing

Reply:
Yeah, this is a good point to note. I think it will be a matter of playing around with the spacing. But I see no reason that can't all be formatted into one p for all three of those.